hsds/util/idUtil.py

```diff
@@ -400,7 +402,7 @@ def getDataNodeUrl(app, obj_id):
     node_count = app["node_count"]
     node_state = app["node_state"]
     if node_state!= "READY" or node_count <= 0 or node_count != len(dn_urls):
-        log.info(f"getDataNodeUrl returning 503 - node_state: {node_state} node count: {node_count}")
```

Comment:
Is there any reason not use f-strings here?

Reply:
No, fixed